src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -116,15 +100,7 @@ public async Task<AnalysisResponse> Analyze(string filePath, string tsConfigFile
                 TSConfigFilePaths = tsConfigFilePaths
             };
 
-            var result = await eslintBridgeProcess.Start();
```

Comment:
I had to revert this change. My original reasoning for this change was that:
1. The process could die and start again on the same port, so the old `if port != currentPort` logic is flawed.
2. This class is the one that knows that a process was spawned, so why shouldn't it return this information back to the callers.

And for JS that worked, `EslintBridgeClient.Analyze` got back the response that this is a new process, threw an exception and init-linter was called.
But for TS, unfortunately TsConfigProvider was called first, which ignored the `IsNewProcess` flag completely. `EslintBridgeClient.Analyze` then received `IsNewProcess=false` and init-linter was not called. This bug happened only after the process died, since the init-linter was always called at least once.